application/forms/Account/ChangePasswordForm.php

```diff
@@ -34,6 +36,17 @@ public function init()
      */
     public function createElements(array $formData)
     {
+        $validators = [];
+
+        try {
+            $helper = new PasswordPolicyHelper();
+            $validators[] = $helper->getPasswordValidator();
+            $helper->addPasswordPolicyDescription($this);
+        } catch (\Throwable $e) {
+            Logger::error($e);
```

Comment:
I would adjust the log message so that it includes the topic, e.g., `"The configured password policy could not be loaded: %s", $e`